build.sbt

```diff
@@ -225,7 +225,7 @@ lazy val rspace = (project in file("rspace"))
   .settings(commonSettings: _*)
   .settings(
     name := "rspace",
-    version := "0.1.1",
+    version := "0.2.1-SNAPSHOT",
```

Comment:
The ticket clearly states `0.2.1` as new version, but I'm wondering if we shouldn't start from `0.2.0`

Reply:
I wondered the same thing, but was told that `0.2.1` would be the first release in the `0.2.x` series...